distributed/stealing.py

```diff
@@ -178,13 +178,13 @@ def transition(
         *args: Any,
         **kwargs: Any,
     ) -> None:
-        if finish == "processing":
-            ts = self.scheduler.tasks[key]
-            self.put_key_in_stealable(ts)
-        elif start == "processing":
+        if start == "processing":
```

Comment:
Please put an in code comment here since this will otherwise die in a refactoring